nengo_viz/static/viz_netgraph.js

```diff
@@ -107,19 +107,37 @@ VIZ.NetGraph = function(parent, args) {
     interact(document.getElementById('main'))
         .on('wheel', function(event) {
             event.preventDefault();
-            if (self.in_zoom_delay) {
-                return;
-            }
-            self.in_zoom_delay = true;
-            setTimeout(function () { self.in_zoom_delay = false; }, 50);
 
             self.menu.hide_any();
 
             var x = (event.clientX / $(self.svg).width())
             var y = (event.clientY / $(self.svg).height());
 
-            var delta = event.wheelDeltaY || -event.deltaY
-            var scale = delta > 0 ? viewport.scale_step_size : 1.0 / viewport.scale_step_size; // will either be 1.1 or ~0.9
+            switch (event.deltaMode) {
+                case 1:  // DOM_DELTA_LINE
+                    if (event.deltaY != 0) {
+                        var delta = Math.log(1. + Math.abs(event.deltaY)) * 60;
+                        if (event.deltaY < 0) {
+                            delta *= -1;
+                        }
+                    } else {
+                        var delta = 0;
+                    }
+                    break;
+                case 2:  // DOM_DELTA_PAGE
+                    // No idea what device would generate scrolling by a page
+                    var delta = 0;
+                    break;
+                case 0:  // DOM_DELTA_PIXEL
+                default:  // Assume pixel if unknown
+                    var delta = event.deltaY;
+                    break;
+            }
```

Comment:
I prefer the original way of writing this.  Partly, this is a style thing that we haven't run into because Python doesn't have switch statements.  But in your (shorter) way of writing things, it isn't clear to me that we expect event.deltaMode==0 as a pretty common case.  So your way is kinda like doing this:

```python
value = 1
if condition:
   value = 2
```

which I find confusing, and I think we've generally been avoiding as a style thing.  So I'd prefer to have the handling for case 0 be explicit inside the switch statement.

As for having both ```default``` and ```case 0```, I liked having both of them in the switch statement, with the comment indicating that we don't really expect ```default``` to happen, but if it does, we'll use the same logic as ```case 0```.